### PR TITLE
chore: drop usage of 'fs-extra' library

### PR DESCRIPTION
**Pull request by alex-gilin**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,23 +1,6 @@
...
-
 jobs: # a collection of steps
-  build-and-test-node10:
-    docker:
-      - image: circleci/node:10.23 # Current BAS node version
-    <<: *default-build
-
-  build-and-test-node12:
-    docker:
-      - image: circleci/node:12 # Future BAS node version
```
Comment (slavik-lvovsky) [inline .circleci/config.yml:18]: check with Rima whether we still need support node12

File: src/utils.ts [hunk 2]
```diff
@@ -52,7 +52,7 @@ export function toText(e: Error): string {
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export function getEnvResources(envFilePath: string): Promise<any> {
 	try {
-		if (fs.existsSync(envFilePath)) {
+		if (existsSync(envFilePath)) {
```
Comment (slavik-lvovsky) [inline src/utils.ts:55]: I think fs.access() is a better choice

File: tests/usage/usageTracker.spec.ts [hunk 3]
```diff
@@ -27,7 +27,7 @@ describe("usageTracker unit tests", () => {
     });
 
     it("ok:: init SWATracker instance", () => {
-        const jsonPackage = parse(fsextra.readFileSync(path.resolve(path.join(__dirname, "..", "..", "package.json")), { encoding: "utf8" }));
+        const jsonPackage = parse(fs.readFileSync(path.resolve(path.join(__dirname, "..", "..", "package.json")), { encoding: "utf8" }));
```
Comment (slavik-lvovsky) [inline tests/usage/usageTracker.spec.ts:30]: you can use fs.promises.readFile, but for test it is also ok

**Review by slavik-lvovsky — APPROVED** [review]

File: src/utils.ts (hunk 2 above)
Comment (alex-gilin) [reply src/utils.ts:55]: fs.access throws exception in case of path not exists.

**Pull request merged**